Dockerfile

```diff
@@ -1,7 +1,14 @@
 # syntax=mcr.microsoft.com/oss/moby/dockerfile:1.3.1
+ARG BUILDERIMAGE="golang:1.17"
+
+ARG ERASERBASEIMAGE="gcr.io/distroless/static:latest"
+ARG MANAGERBASEIMAGE="gcr.io/distroless/static:nonroot"
+
+ARG TARGETOS
```

Comment:
These are essentially unused.
They need to be set within the `FROM` that they are going to be consumed in.

Reply:
omg, thanks for catching that!